apps/app/src/app/redirect/redirect.component.ts

```diff
@@ -1,18 +1,67 @@
 import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { SDKService } from '@flaps/auth';
-import { take } from 'rxjs';
+import { injectScript } from '@flaps/core';
+import { take, tap, concatMapTo } from 'rxjs';
 
+declare var gapi: any;
+declare var google: any;
+
+const SCOPES = 'https://www.googleapis.com/auth/drive.metadata.readonly https://www.googleapis.com/auth/drive.readonly';
+const DISCOVERY_DOCS = ['https://www.googleapis.com/discovery/v1/apis/drive/v3/rest'];
 @Component({
   template: 'Redirecting to Nuclia Desktop…',
 })
 export class RedirectComponent {
   constructor(private sdk: SDKService, private route: ActivatedRoute) {
     this.route.queryParams.pipe(take(1)).subscribe((params) => {
       let redirectUrl: string = params['redirect'] || '';
```

Comment:
what do you think about exit if param['redirect'] is different than 'nuclia-desktop://' or 'localhost' ?
Maybe someone could create malicious urls like 'nuclia.cloud/redirect?redirect=maliciuos.com'

Reply:
waouh that's a very very good point!!!

Reply:
fixed